src/bio2bel/sources/rhea.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+"""Convert Rhea to BEL."""
+
+import logging
+from typing import Any, Dict, List, Tuple
+
+import rdflib
+
+import pybel
+import pybel.dsl as dsl
+from bio2bel.constants import BIO2BEL_MODULE
+
+__all__ = [
+    'get_bel',
+]
+
+logger = logging.getLogger(__name__)
+
+MODULE_NAME = 'rhea'
+VERSION = '116'  # released on 2020-12-02
```

Comment:
would be nice to be more dynamic about this. You can do
```python
import bioversions
VERSION = bioversions.get_version('rhea')
```
to get the current version